src/components/container/EhrNewsFeed/EhrNewsFeed.tsx

```diff
@@ -164,10 +160,8 @@ export default function (props: EhrNewsFeedProps) {
 
 function NewsFeedListItem(props: { event: EhrNewsFeedEventModel, onClick: (event: EhrNewsFeedEventModel) => void }) {
     let definition = eventTypeDefinitions[props.event.Type];
-    let date = formatDateForLocale(props.event.Date, "h:mm a");
-    if (date === "12:00 AM") {
```

Comment:
The new helper automatically returns an empty time string if the time is 00:00:00, so we no longer need this specific check.